os/skia/skia_surface.h

```diff
@@ -115,6 +118,9 @@ class SkiaSurface final : public Surface {
   static SurfaceRef loadSurface(const char* filename);
 
 private:
+  static SkColorType deductSkColorType(const os::SurfaceFormatData& sf);
+  static os::PixelAlpha asPixelAlpha(const SkAlphaType& at);
+  static SkAlphaType asSkAlphaType(const os::PixelAlpha& pa);
```

Comment:
All functions that receive an enum, can receive it as a value instead of `const T&`, because it's an integer.
```suggestion
  static os::PixelAlpha asPixelAlpha(SkAlphaType at);
  static SkAlphaType asSkAlphaType(os::PixelAlpha pa);
```
e.g. Without this change you cannot use asPîxelAlpha() with a literal value.